carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -1046,16 +1046,22 @@ void WMBroadcaster::geofenceCallback(const cav_msgs::TrafficControlMessage& geof
 {
   
   std::lock_guard<std::mutex> guard(map_mutex_);
+  std::stringstream reason_ss;
   // quickly check if the id has been added
   if (geofence_msg.choice != cav_msgs::TrafficControlMessage::TCMV01) {
-    ROS_WARN_STREAM("Dropping received geofence for unsupported TrafficControl version: " << geofence_msg.choice);
+    reason_ss << "Dropping received geofence for unsupported TrafficControl version: " << geofence_msg.choice;
+    ROS_WARN_STREAM(reason_ss.str());
```

Comment:
I could be wrong but I think since you are using the _STREAM logs you can just pass in the stream object directly
```
ROS_WARN_STREAM(reason_ss);
```
Which avoids the extra string conversion

Reply:
There seems to be some complains from CI.